src/test/java/com/exadel/discountwebapp/category/service/CategoryServiceIntegrationTest.java

```diff
@@ -5,20 +5,17 @@
 import com.exadel.discountwebapp.category.vo.CategoryRequestVO;
 import com.exadel.discountwebapp.category.vo.CategoryResponseVO;
 import com.exadel.discountwebapp.exception.EntityNotFoundException;
-import org.junit.runner.RunWith;
+import org.assertj.core.util.Lists;
```

Comment:
?

Reply:
Do you think it's necessary to add a new dependency for one-line change?
There is in-build method in org.assertj.core.util.Lists. 
It looks useful.

Reply:
Changed to Guava